src/main_window.py

```diff
@@ -1612,7 +1613,11 @@ def logout(self, action, param):
                 os.system("dbus-send --print-reply --session --dest=org.kde.LogoutPrompt /LogoutPrompt org.kde.LogoutPrompt.promptLogout")
             elif self.win.environment == 'COSMIC (New)':
                 os.system("dbus-send --print-reply --session --dest=com.system76.CosmicSession --type=method_call /com/system76/CosmicSession com.system76.CosmicSession.Exit")
+            elif self.win.environment == 'Hyprland':
+                os.system("hyprctl dispatch exit") #does logout without prompting the user because i couldn't find a good way to do it
+		#needs to be fixed for flatpak unfortunately I do not know how to make this call on flatpak via Python
             else:
+                print(f'{self.win.environment} is not found in our database it would be of great value if you would leave a issue on our github page: https://github.com/vikdevelop/SaveDesktop')
```

Comment:
```suggestion
```
Since most DEs except KDE, Cosmic (Rust version) and Xfce use org.gnome.SessionManager, there is no need to add this message here.

Reply:
okay i will remove it in my next commit

Reply:
done